src/types/ErrorTypesForm.ts

```diff
@@ -0,0 +1,6 @@
+export enum ErrorForm {
+  NAME = 'Name is required',
+  EMAIL = 'Email is required',
+  COMMENT = 'Enter some text',
+  INVALID_EMAIL = 'Enter correct e-mail',
```

Comment:
The error message for INVALID_EMAIL should be more professional, instead of 'Enter correct e-mail', it could be 'Please enter a valid email address.'